src/examples/algorithms/AnomalyDetection.cpp

```diff
@@ -0,0 +1,186 @@
+#include <iostream>
```

Comment:
Also add a doc string to the file describing what it does and example command line usage.